rollup.config.mjs

```diff
@@ -45,11 +49,8 @@ export default [
       resolve(),
       json(),
       commonjs(),
-      babel({
-        babelrc: false,
-        presets: ['@babel/preset-env'],
-      }),
-      sizeCheck(),
+      babel(babelConfig),
+      // sizeCheck(),
```

Comment:
`rollup-plugin-filesize-check` throws `[!] (plugin filesize) TypeError: The "string" argument must be of type string or an instance of Buffer or ArrayBuffer. Received undefined` when handling sourcemaps, so have commented out for now.

Reply:
cool, feel-free to remove filesize-check if it's causing problems. rollup is full of these sorta gotchas.